CONTRIBUTING.md

```diff
@@ -146,11 +148,16 @@ summon ./test_e2e
    from your local machine).
 
 When releasing a new version of the Service Broker, you will need to include a
```

Comment:
This is outside of your change, but should `you will need to include` be changed to `you will need to upload`?